src/test/java/com/mongodb/jdbc/MongoSQLResultSetTest.java

```diff
@@ -52,7 +52,7 @@
     static {
         try {
             schema = generateMongoJsonSchema();
-            resultSetMetaData = new MongoSQLResultSetMetaData(schema);
+            resultSetMetaData = new MongoSQLResultSetMetaData(schema, false);
```

Comment:
How did you determine `false` vs `true` here? For this one, I don't see us using `resultSetMetaData` anywhere in the tests, so should we remove it? Or is its construction implicitly testing something?